pkg/data/util.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package data
+
+// hashString computes the hash of s.
+func hashString(s string) uint32 {
```

Comment:
Do we need to write the hash by ourselves? There are some hash packages.

Reply:
https://github.com/google/uuid

Reply:
Changed to starlark's string hash